templates/pages/signin.html

```diff
@@ -48,12 +44,12 @@ <h1 class="login-form-title">Bem-vindo de volta!</h1>
       <a class="forgot-password-link" href="{% url 'forgot_password' %}"
```

Comment:
Nome da url incorreto
```suggestion
      <a class="forgot-password-link" href="{% url 'forgot-password' %}"
```